### PR TITLE
chore: drop promptui dependency, replace with survey

### PR DESCRIPTION
**Pull request by zlesnr**

### CODE REVIEW

**Review by sanderblue — APPROVED** [review]
🐑 it

File: internal/install/execution/go_task_recipe_executor.go [hunk 1]
```diff
@@ -233,26 +234,31 @@ func varFromPrompt(envConfig recipes.VariableConfig) (string, error) {
...
-	}
+	value := ""
+	var prompt survey.Prompt
 
 	if envConfig.Secret {
-		prompt.HideEntered = true
-		prompt.Mask = '*'
+		prompt = &survey.Password{
+			Message: msg,
+		}
```
Comment (jbeveland27) [inline internal/install/execution/go_task_recipe_executor.go:243]: Does this automagically mask the input with `*****`?

Comment (zlesnr) [reply internal/install/execution/go_task_recipe_executor.go:243]: It does.  You can see an example here: https://github.com/AlecAivazis/survey/#password

Comment (zlesnr) [reply internal/install/execution/go_task_recipe_executor.go:243]: Feel free to check the branch out and give it a test though.

Comment (jbeveland27) [reply internal/install/execution/go_task_recipe_executor.go:243]: Ah cool 🙌🏼

File: internal/install/execution/go_task_recipe_executor.go [hunk 2]
```diff
@@ -233,26 +234,31 @@ func varFromPrompt(envConfig recipes.VariableConfig) (string, error) {
...
+		defaultValue := ""
+
+		if envConfig.Default != "" {
+			defaultValue = envConfig.Default
+		}
+
+		prompt = &survey.Input{
+			Message: msg,
+			Default: defaultValue,
+		}
```
Comment (jbeveland27) [inline internal/install/execution/go_task_recipe_executor.go:254]: Would be interesting to consider using the [suggest options](https://github.com/AlecAivazis/survey#suggestion-options) feature vs what we do now with having the default as a part of the [prompt message](https://github.com/newrelic/open-install-library/blob/main/recipes/newrelic/infrastructure/ohi/apache/debian.yml#L40)

Comment (jbeveland27) [reply internal/install/execution/go_task_recipe_executor.go:254]: That'd involve a change to the schema, so wouldn't happen right away. But might offer a better UX.

Comment (zlesnr) [reply internal/install/execution/go_task_recipe_executor.go:254]: Yeah, that sounds nice.  Perhaps in another pr?

Comment (jbeveland27) [reply internal/install/execution/go_task_recipe_executor.go:254]: Yup, not needed for this one, but possible future enhancement.

**Review by jbeveland27 — APPROVED** [review]
LGTM 🚢

**Pull request merged**